contracts/vesting.token/src/vesting.token.cpp

```diff
@@ -0,0 +1,101 @@
+#include <vesting.token/vesting.token.hpp>
+
+namespace vestingtoken {
+
+    void vestingToken::updatedate(eosio::time_point sales_date, eosio::time_point launch_date) {
```

Comment:
can we not use `time_point_sec::from_is_string()` here? and the arguments are then strings?